app/Nova/Resources/UdbOrganizer.php

```diff
@@ -119,8 +128,14 @@ public function fields(NovaRequest $request): array
             Text::make('UiTPAS', static function (UdbOrganizerModel $model) {
                 /** @var IntegrationRepository $integrationRepository */
                 $integrationRepository = App::get(IntegrationRepository::class);
-                $integration = $integrationRepository->getById($model->toDomain()->integrationId);
-                $keycloakClient = $integration->getKeycloakClientByEnv(Environment::Production);
+
+                try {
+                    $integration = $integrationRepository->getById($model->toDomain()->integrationId);
+                    // Sometimes because issues on the keycloak side the Prod keys are not generated, breaking the entire admin udb organizer screen
```

Comment:
We are not sure if this is Keycloak related! As we found inside our code, we don't handle refresh tokens in the Horizon worker!